packages/blockchain-wallet-v4-frontend/src/modals/Swap/SuccessfulSwap/index.tsx

```diff
@@ -18,46 +20,156 @@ const Wrapper = styled(FlyoutWrapper)`
   flex-direction: column;
 `
 
+const StyledText = styled(Text)`
+  margin-top: 24px;
```

Comment:
Instead of creating all these new styled components can you just use the <Padding /> component from constellation?

Reply:
https://blockchain.github.io/blockchain-wallet-v4-frontend/?path=/story/structural-padding-intro--page

Reply:
Mmm... tried but it seems that the wallet isn't using the latest version of the library and it's not exporting it. Will leave it as it is now. thanks anyway.

![image](https://user-images.githubusercontent.com/97299628/171935356-db5f56bd-2e18-48e8-92ed-86408525dd76.png)